app/experimenter/nimbus-ui/src/components/PageResults/ExternalConfigAlert/index.tsx

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { Alert } from "react-bootstrap";
+import { MetadataExternalConfig } from "../../../lib/visualization/types";
+import LinkExternal from "../../LinkExternal";
+
+export const ExternalConfigAlert = ({
+  externalConfig,
+}: {
+  externalConfig: MetadataExternalConfig;
+}) => (
+  <Alert variant="warning" data-testid="external-config-alert">
+    <Alert.Heading>Analysis has manual overrides in Jetstream</Alert.Heading>
+    <p>
+      The results shown on this page are from an analysis ran with at least one
+      experiment override that affects only the <em>analysis</em>, meaning{" "}
+      <strong>
+        experiment data on the Summary page will not reflect this.
+      </strong>
+    </p>
+    <ul className="pl-0">
+      Overrides (
+      <LinkExternal href={externalConfig.url} data-testid="external-config-url">
+        click here
+      </LinkExternal>{" "}
+      to view the config file):
+      {externalConfig.reference_branch && (
```

Comment:
Will there be any other external config properties mentioned here besides `reference_branch`? The `MetadataExternalConfig` type also includes these:
```
  end_date: string | null;
  enrollment_period: string | null;
  skip: boolean;
  start_date: string | null;
```

Reply:
This PR is just for `reference_branch`, I noted the separate PR bit in the PR description / issue number in [a comment here](https://github.com/mozilla/experimenter/pull/6264/files#diff-1255a403cd47ad0be32d2918921c64471bdcf96d6808ffee9487be30fa0e815fR50) 😁